Volume/Views/DebriefArticleView.swift

```diff
@@ -74,20 +86,28 @@ struct DebriefArticleView: View {
             Header(header, .center)
                 .padding(.top, 24)
             
-            VStack(spacing: 16) {
-                if let url = article.imageUrl {
-                    WebImage(url: url)
-                        .resizable()
-                        .grayBackground()
-                        .aspectRatio(contentMode: .fill)
-                        .frame(width: 275, height: 275)
-                        .clipped()
-                } else {
-                    WebImage(url: article.publication.profileImageUrl)
-                        .resizable()
-                        .grayBackground()
-                        .frame(width: 275, height: 275)
-                        .padding(30)
+            NavigationLink(destination: BrowserView(initType: .readyForDisplay(article), navigationSource: .weeklyDebrief)) {
+                VStack(spacing: 16) {
+                    if let url = article.imageUrl {
+                        WebImage(url: url)
+                            .resizable()
+                            .grayBackground()
+                            .aspectRatio(contentMode: .fill)
+                            .frame(width: 275, height: 275)
+                            .clipped()
+                    } else {
+                        WebImage(url: article.publication.profileImageUrl)
+                            .resizable()
+                            .grayBackground()
+                            .frame(width: 275, height: 275)
```

Comment:
factor out the constants into a variable eg.`.frame(width: frameWidth, height: frameHeight)`